data/net/minecraft/world/entity/Mob.mapping

```diff
@@ -85,9 +85,9 @@ CLASS net/minecraft/world/entity/Mob
 	METHOD getItemBySlot (Lnet/minecraft/world/entity/EquipmentSlot;)Lnet/minecraft/world/item/ItemStack;
 		ARG 1 slot
 	METHOD getMaxFallDistance ()I
-		COMMENT The maximum height from where the entity is alowed to jump (used in pathfinder)
+		COMMENT The maximum height from where the entity is allowed to jump (used in pathfinder)
 	METHOD getMaxHeadXRot ()I
-		COMMENT The speed it takes to move the entityliving's head rotation through the faceEntity method.
+		COMMENT The speed it takes to move the entitylivings head rotation through the faceEntity method.
```

Comment:
Again I think the original is correct here. 
May as well also change it to `mob` rather than `entityliving`.